test/lit/passes/stack-check_enable-mutable-globals.wast

```diff
@@ -0,0 +1,45 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
```

Comment:
The name of this file doesn't need to include the full string `enable-mutable-globals` anymore.   Perhaps just call it `stack-check-mutable-globals`?